ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/network-services-l3-port-channels.yml

```diff
@@ -167,19 +168,16 @@ tenants:
               point_to_point: false
               area: "0.0.0.0"
               cost: 1
-              authentication: "message-digest"
+              authentication: message-digest
               message_digest_keys:
                 - id: 1
                   hash_algorithm: "sha256"
-                  key: testpass1
+                  key: ocAzkT2JRNKrbPGT790Geg==
                 - id: 2
-                  key: testpass2
-                  # Test when 'id' and 'key' are not set under message-digest keys.
-                  # TODO: AVD 6.0.0 Make 'id' a primary key and 'key' required
```

Comment:
This TODO addresses the discussion about id being primary key. We already decided to push to 6.0 it seems.